.circleci/config.yml

```diff
@@ -4,11 +4,20 @@ ansibleIntegrationTest: &ansibleIntegrationTest
     steps:
       - checkout
       - setup_remote_docker
+      - run:
+          description: upgrade pip
+          command: pip install --upgrade pip
```

Comment:
Can we move all the install part to the image build process? This way the builds here are shorter, less error-prone and more controlled, wdyt?